### DISCUSSION TITLE
Does Dagster expose any observability metrics?

### DISCUSSION THREAD

**Discussion by andrewm4894** [dagster-io/dagster#17950] [Q&A] [answered]
Is there an endpoint anywhere where Dagster exposes something like prometheus or statsd type metrics about Dagster? 

I am interested in being able to monitor and instrument my running dagster app.

**Comment by alangenfeld** [comment, marked as answer]
Not currently, here is probably the relevant issue to chime in on https://github.com/dagster-io/dagster/issues/11191
